packages/block-editor/src/store/selectors.js

```diff
@@ -1417,3 +1417,14 @@ function getReusableBlocks( state ) {
 export function isNavigationMode( state ) {
 	return state.isNavigationMode;
 }
+
+/**
+ * Returns whether block editor can search and install uninstalled blocks.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {boolean}     Whether the downloadable blocks feature is enabled.
+ */
+export function getIsDownloadableBlocksEnabled( state ) {
```

Comment:
Nit: this combination of get + is might be nicer to read as:
```suggestion
export function isDownloadableBlocksEnabled( state ) {
```